stable/spring-cloud-data-flow/README.md

```diff
@@ -101,6 +108,19 @@ The following tables list the configurable parameters and their default values.
 
 ### MySQL Configuration
 
-| Parameter                  | Description           | Default                   |
-| -------------------------- | --------------------- | ------------------------- |
-| mysql.mysqlDatabase        | MySQL database name   | dataflow
+| Parameter                  | Description                  | Default                   |
+| -------------------------- | ---------------------------- | ------------------------- |
+| mysql.enabled              | Enable deployment of MySQL   | true
+| mysql.mysqlDatabase        | MySQL database name          | dataflow
+
+### External MySQL Configuration (mysql.enabled=false)
```

Comment:
I disagree with this, many other charts use the same convention. 

https://github.com/helm/charts/tree/master/stable/wordpress
https://github.com/helm/charts/tree/master/stable/prestashop
https://github.com/helm/charts/tree/master/stable/artifactory

Reply:
not debating if this convention is used to enable or disable charts, that's understood. im debating how its used in the context of this line. in short it would likely be better to make a note about `mysql.enable` needing to be false in the content rather than the header.